src/Dapr.AspNetCore/TopicAttribute.cs

```diff
@@ -24,6 +24,7 @@ public TopicAttribute(string pubsubName, string name)
 
             this.Name = name;
             this.PubsubName = pubsubName;
+            this.Priority = int.MaxValue;
```

Comment:
I would prefer this to be initialized to zero unless there's a reason why there can't be. ASP.NET core's use of priorities/orders works this way.

Reply:
- lower is better
- zero is the default

Reply:
I have it initialize to MaxValue so that it comes after the routed attributes in the OrberBy I do in DaprEndpointRouteBuilderExtensions.cs, but the way the code is written now, it doesn't really matter that much. If it gets ordered after routed attributes, then it will be assigned to `Routes.Default` instead of `Route` (which then has be be moved to `Routes.Default` when a routed attribute is encountered. TLDR is a tiny optimization.